pyproject.toml

```diff
@@ -13,6 +13,8 @@ python = ">=3.9,<3.12"
 PySide6 = "^6.4.0"
 platformdirs = "^2.5.4"
 matplotlib = "^3.6.2"
+pypubsub = "^4.0.3"
+pyserial = "^3.5"
```

Comment:
I'm a bit confused by this change. It looks like you added this explicitly in d4c9452f3640aa1de22d25059bc2c64af9a407c2 but the changelog doesn't say why you're adding dependencies (I know they've already been added on `main` though). Were you trying to prevent a merge conflict?

Reply:
Yes this was to try and deal with the merge conflict 😅

Reply:
Ah I see. It looks like you didn't actually merge in `main` though... right? To fix the conflicts you do actually need to do a merge operation otherwise you just confuse git and get more merge conflicts. (The other thing is that if you're merging in changes from somewhere else it's nice to have a commit saying this has happened.)

You will need to do it locally because you'll need to manually regenerate `poetry.lock` when you're done. Lemme know if you get stuck.